src/realtime_ui/trajectory.css

```
@@ -46,8 +47,13 @@ button { color: inherit; }
 .span.selected { z-index: 2; opacity: 1; box-shadow: 0 0 0 1px #fff, 0 0 0 2px var(--blue); }
 .span.filtered { opacity: .12; }
 .turn-boundary { position: absolute; top: -8px; bottom: -8px; width: 1px; background: #d9dce1; pointer-events: none; }
-.split { display: flex; flex: 1; min-height: 0; overflow: hidden; }
-.ledger { flex: 1; min-width: 280px; overflow: auto; }
+.split { position: relative; display: flex; flex: 1; min-height: 0; overflow: hidden; }
+.ledger-shell { position: relative; flex: 1; min-width: 280px; overflow: hidden; }
+.ledger { height: 100%; overflow: auto; }
+.scroll-bottom { position: absolute; right: 16px; bottom: 16px; display: grid; width: 32px; height: 32px; padding: 0; place-items: center; border: 1px solid var(--line); border-radius: 50%; background: var(--bg); box-shadow: 0 2px 9px rgb(0 0 0 / 14%); color: var(--muted); cursor: pointer; opacity: .9; }
+.scroll-bottom:hover { color: var(--text); opacity: 1; }
+.scroll-bottom[hidden] { display: none; }
+.scroll-bottom svg { width: 17px; height: 17px; fill: none; stroke: currentColor; stroke-width: 1.7; stroke-linecap: round; stroke-linejoin: round; }
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)